cmd/podman/images/list.go

```diff
@@ -170,14 +174,18 @@ func writeTemplate(imgs []imageReporter) error {
 	return tmpl.Execute(w, imgs)
 }
 
-func sortImages(imageS []*entities.ImageSummary) []imageReporter {
+func sortImages(imageS []*entities.ImageSummary) ([]imageReporter, error) {
 	imgs := make([]imageReporter, 0, len(imageS))
+	var err error = nil
 	for _, e := range imageS {
 		var h imageReporter
 		if len(e.RepoTags) > 0 {
 			for _, tag := range e.RepoTags {
 				h.ImageSummary = *e
-				h.Repository, h.Tag = tokenRepoTag(tag)
+				h.Repository, h.Tag, err = tokenRepoTag(tag)
+				if err != nil {
+					return nil, err
```

Comment:
We could give some more information here:
```golang
return nil, errors.Wrapf(err, "error parsing repository tag %q:, tag)
```
Now we make sure that the "bad" string is part of the error message.